### PR TITLE
SAK-43316 Samigo Manage Rubrics error for instructors without permission

### PR DESCRIPTION
**Pull request by bgarciaentornos**

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/AssessmentFacadeQueries.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=5074879&bid=16696682)

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AuthorBean.java [hunk 1]
```diff
@@ -963,11 +964,15 @@ public void setGroupFilterEnabled(boolean groupFilterEnabled) {
...
-			return Boolean.TRUE;
+		try {
+			if(!isPublished && rubricsService.hasAssociatedRubric(RubricsConstants.RBCS_TOOL_SAMIGO, assessmentId + "." + questionId)){
+				return Boolean.TRUE;
+			}
+			if(isPublished && rubricsService.hasAssociatedRubric(RubricsConstants.RBCS_TOOL_SAMIGO, RubricsConstants.RBCS_PUBLISHED_ASSESSMENT_ENTITY_PREFIX + assessmentId + "." + questionId)){
+				return Boolean.TRUE;
+			}
+		} catch(HttpClientErrorException hcee) {
+			log.debug("Current user doesn't have permission to get a rubric");
```
Comment (ern) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AuthorBean.java]: log.debug should print the exception message i.e. 
`log.debug("Current user doesn't have permission to get a rubric, {}" hcee.getMEssage());`

Comment (bgarciaentornos) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AuthorBean.java]: I'll add it, but I believe it was just 'null'

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/AssessmentFacadeQueries.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=5074879&bid=16696682)

**Pull request merged**